main/src/CaelumSystem.cpp

```diff
@@ -36,7 +36,6 @@ namespace Caelum
             LogManager::getSingleton().logMessage ("Caelum: Plugin not installed; installing now.");
             new CaelumPlugin ();
             CaelumPlugin::getSingletonPtr ()->install ();
-            CaelumPlugin::getSingletonPtr ()->initialise ();
```

Comment:
why did you remove this call?

Reply:
Install calls already initialise in his body, this line generate a double call and with the setting of mIsInstalled = true, the second call  triggers the assert